device/celestica/x86_64-cel_silverstone-r0/led_proc_init.soc

```diff
@@ -0,0 +1,12 @@
+#Enable all ports
+sleep 6
```

Comment:
what is this sleep for?

Reply:
The two line should be deleted

Reply:
Please update the PR with this change.

Reply:
fixed in https://github.com/SONIC-DEV/sonic-buildimage/commit/2294129ca65505c513cce98386a796a7af3373ed

Reply:
fixed in https://github.com/Azure/sonic-buildimage/pull/3474/commits/2294129ca65505c513cce98386a796a7af3373ed